codespell_lib/_codespell.py

```diff
@@ -697,10 +731,8 @@ def main(*args):
             return EX_USAGE
         build_ignore_words(ignore_words_file, ignore_words)
 
-    ignore_words_list = options.ignore_words_list or []
-    for comma_separated_words in ignore_words_list:
-        for word in comma_separated_words.split(','):
-            ignore_words.add(word.strip())
+    uri_regex = re.compile(uri_regex_def)
```

Comment:
Can you add a try/catch around it like the others and log the failure.

Reply:
Done

Reply:
Thanks. Yeah I'm personally generally a fan of not baking config like this in, but providing a sane default e.g. if someone wants a different protocol such as TOR or svn, or wants a more or less restrictive regex like that site lists. Plus with the config file stuff for options we've got now, it's easy for them to just set it once in their repo and forget about it.

Reply:
Now done.